pulsar-websocket/src/main/java/org/apache/pulsar/websocket/ConsumerHandler.java

```diff
@@ -299,8 +301,7 @@ private void checkResumeReceive() {
 
     private void handleAck(ConsumerCommand command) throws IOException {
         // We should have received an ack
-        TopicMessageId msgId = new TopicMessageIdImpl(topic.toString(),
-                (MessageIdAdv) MessageId.fromByteArray(Base64.getDecoder().decode(command.messageId)));
+        MessageId msgId = MessageId.fromByteArray(Base64.getDecoder().decode(command.messageId));
```

Comment:
WE don't need to change it, isn't it?

Reply:
we need to change this, because if we subscribe the multi-topic, the `topic` is not unique.